### PR TITLE
Create contribution documentation

### PR DESCRIPTION
**Pull request by kristofferlund**
#60

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+<!--
+  Thanks for opening an issue! A few things to keep in mind:
+  
+  - The issue tracker is only for bugs and feature requests.
+  - Before reporting a bug, please make sure you have the latest version of Webviz and try to do a clean install of the project.
```
Comment (sivhollup) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Also add something along the lines of: , ensure the bug has not been reported before
Or is that our responsibility?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+* How to reproduce
+
+##Pull Requests
+
+
+### Step 1: Fork
+Fork the project [on Github](https://github.com/Statoil/webviz) and clone your fork locally
+```sh
+$ git clone git@github.com:username/webviz.git
+$ cd electron
```
Comment (anders-kiaer) [inline CONTRIBUTING.md]: `cd webviz`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+[error codes](http://pycodestyle.pycqa.org/en/latest/intro.html#error-codes).
+
+##### Testing
+The testing is done with [Coverage](https://coverage.readthedocs.io/en/coverage-4.5.1a/), and 
+the specified test rate for each component is 100%. If you for some reason want to reduce the 
+test rate, this has to be argued for, and the reviewer will have to decide if it makes sense 
+to lower the treshold. 
+
+### Step 5: Commit
+Add the changed files to a commit. _Note: package.lock files should not be included, as they
```
Comment (anders-kiaer) [inline CONTRIBUTING.md]: `package.lock` → `package-lock.json`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+are updated in a dedicated pull request._
+
+```sh
+$ git add my/changed/files
+$ git commit -m "Describing message about what you have done"
+```
+Please be specific when writing the pull request message. You can also tagg your message
```
Comment (anders-kiaer) [inline CONTRIBUTING.md]: tagg → tag

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+with the issue you are solving by putting the issue number in a hashtag at the end
+of the commit message. _Note: this __has__ to be at the end of the message, as everything 
+else after the hashag gets escaped_
```
Comment (anders-kiaer) [inline CONTRIBUTING.md]: hashag → hashtag

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (anders-kiaer) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I think it is fair to require a quick search before creating a new issue. Maybe something along the lines of:

> Before opening a new issue, please search for similar issue among existing and closed issues.

**Pull request merged**